tutorial/lesson_09_update_definitions.cpp

```diff
@@ -208,6 +208,7 @@ int main(int argc, char **argv) {
         Buffer<int> halide_result = histogram.realize({256});
 
         // The equivalent C is:
+        // NOLINTBEGIN
```

Comment:
I'd really rather not have linter line noise in tutorials. It would be better to not lint them

Reply:
I went the less-lazy route and created separate settings for our tutorials. That way, we still get some coverage, but it doesn't apply to the rest of the codebase (and doesn't apply transitively to our headers).